### DISCUSSION TITLE
Please Upgrade Python Version to 3.10.x

### DISCUSSION THREAD

**Discussion by kaif-00z** [vercel/vercel#9031] [Ideas] [answered]
as we know python 3.9 in old and with 3.10 some syntax changes and upgrades are there... 

So I want Vercel To Upgrade python version ASAP..

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out @kaif-00z! The Python 3.10 Lambda runtime is not available yet on AWS. Once it becomes available, we will enable it for Vercel deployments. You can learn more in https://github.com/vercel/community/discussions/639

**Reply by kirmola** [reply]
[This](https://aws.amazon.com/about-aws/whats-new/2023/07/aws-lambda-python-3-11/) article from AWS is saying they have python 3.11 available.


When it's planned to be added on vercel @amyegan

**Reply by amyegan** [reply]
I don't have an ETA for full automatic build support, but there is a workaround available. We currently have support via the CLI –https://github.com/vercel/community/discussions/639#discussioncomment-7821041
